src/generators/gencs.ml

```diff
@@ -2261,7 +2261,7 @@ let generate con =
 						| overloads -> overloads
 					in
 					List.iter (fun cf ->
-						if (has_class_flag cl CInterface) || cf.cf_expr <> None then
+						if (has_class_flag cl CInterface) || (has_class_flag cl CAbstract) || cf.cf_expr <> None then
```

Comment:
I think we should aim to generate native abstract classes where supported. But that could be done later.

Reply:
Actually, I think we have to do it right away. Otherwise we run into problems if an abstract method has a return type since we can't just generate an empty block for it...

Reply:
Ah, ok